pkgs/tools/filesystems/sasquatch/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, fetchurl
+, lib
+, lz4 ? null
+, lz4Support ? false
+, lzma
+, lzo
+, stdenv
+, wget
+, xz
+, zlib
+}:
+
+let
+  patch = fetchFromGitHub {
+    owner = "devttys0";
+    repo = "sasquatch";
+    rev = "3e0cc40fc6dbe32bd3a5e6c553b3320d5d91ceed";
+    sha256 = "19lhndjv7v9w6nmszry63zh5rqii9v7wvsbpc2n6q606hyz955g2";
+  } + "/patches/patch0.txt";
+in
+stdenv.mkDerivation rec {
+  name = "sasquatch";
+
+  src = fetchurl {
```

Comment:
Does this work with the version of `squashfs` we have in nixpkgs (`4.4dev_20180612`)?